examples/common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -191,19 +171,15 @@ def __process_work_order_submission(self, wo_id, input_json_str, response):
             self.workorder_list.append(wo_id)
             self.workorder_count += 1
 
-            response = utility.create_error_response(
+            raise JSONRPCDispatchException(
```

Comment:
Do we need to add an explicit try/catch block to handle this exception or will this be handled by JSONRPCResponseManager in tcs_listener.py. If so, its good to add a comment in code as its not very obvious how this exception is handled

Reply:
The comment has been added to class level of `TCSWorkOrderHandler` in this file~ @manojgop